services/account_type/resource_owner.yaml

```diff
@@ -15,10 +15,17 @@ Parameters:
     Type: String
     Description: |
       CloudZero AWS AccountID; for cross-account Role Access
+    Default: "061190967865"
   IsResourceOwnerAccount:
     Type: String
     Description: |
       Should this template instantiate any resources?
+  ResourceOwnerRoleNameParam:
+    Type: String
+    Description: |
+      Name of the Role to be created in the resource account. This must be identical
+      in every resource account.
+    Default: "CloudzeroResourceOwnerRole"
```

Comment:
Rename to `cloudzero-access-role`? We would one day want to standardize the role name to one thing across all accounts (payer or resource).